File: Lab4/pliki/zad3/Product.cs
namespace MateuszLopacinskiEFProducts
{
  internal class Product
  {
    public int ProductID { get; set; }
    public string ProductName { get; set; }
    public int UnitsOnStock { get; set; }

    public override string ToString()
    {
      return $"{ProductName} ({UnitsOnStock} szt.)";
    }
  }
}

File: Lab5/3/Z tabelą łącznikową/Product.java
package com.matipl01;

import javax.persistence.*;

@Table(name = "Products")
@Entity
@SequenceGenerator(name = "Product_SEQ")
public class Product {
    @Id
    @GeneratedValue(strategy=GenerationType.SEQUENCE, generator = "Product_SEQ")
    private int productID;

    private String productName;
    private int unitsInStock;

    public Product() {}

    public Product(String productName, int unitsInStock) {
        this.productName = productName;
        this.unitsInStock = unitsInStock;
    }

    @Override
    public String toString() {
        return productName + " (" + unitsInStock + " szt.)";
    }
}
